### DISCUSSION TITLE
Could be nice to choose icons for tools, otherwise most of them have terminal icon

### DISCUSSION THREAD

**Discussion by zs-dima** [mikekazakov/nimble-commander#407] [Ideas]
Could be nice to choose icons for tools, otherwise most of them have terminal icon

**Comment by mikekazakov** [comment]
Agree, external tools can have an additional field pointing to an icon to show in case there's no good default. I'm not sure when or if ever I'll be able to find time to implement it, so you'd like to have this feature in - please consider contributing to the project.
